src/types/publication.types.ts

```diff
@@ -114,108 +123,13 @@ export interface PagesMonograph extends BackendType {
   pages: string;
 }
 
-interface PublicationInstance {
-  type: JournalType | ReportType | DegreeType | BookType | '';
-  articleNumber: string;
-  issue: string;
-  pages: PagesRange | PagesMonograph | null;
-  peerReviewed: boolean;
-  volume: string;
-}
-
-interface PublicationContext {
-  type: PublicationType | '';
-  level: string | number | null;
-  onlineIssn: string;
-  openAccess: boolean;
-  peerReviewed: boolean;
-  title: string;
-  url?: string;
-  seriesTitle?: string;
-}
-
-interface PublicationReference extends BackendType {
-  doi: string;
-  publicationInstance: PublicationInstance;
-  publicationContext: PublicationContext;
-}
-
-interface PublicationEntityDescription extends BackendType {
-  mainTitle: string;
-  abstract: string;
-  description: string;
-  tags: string[];
-  npiSubjectHeading: string;
-  date: PublicationDate;
-  language: LanguageValues;
-  contributors: Contributor[];
-  isbn: string;
-  numberOfPages: string;
-  series: Publisher;
-  specialization: string;
-  textBook: boolean;
-  reference: PublicationReference;
-}
-
-const emptyDate: PublicationDate = {
-  type: BackendTypeNames.PUBLICATION_DATE,
-  year: '',
-  month: '',
-  day: '',
-};
-
-const emptyPublicationInstance: PublicationInstance = {
-  type: '',
-  volume: '',
-  issue: '',
-  articleNumber: '',
-  pages: null,
-  peerReviewed: false,
-};
-
-const emptyPublicationContext: PublicationContext = {
-  type: '',
-  level: '',
-  onlineIssn: '',
-  openAccess: false,
-  peerReviewed: false,
-  title: '',
-  url: '',
-  seriesTitle: '',
-};
-
-const emptyReference: PublicationReference = {
-  type: BackendTypeNames.REFERENCE,
-  doi: '',
-  publicationInstance: emptyPublicationInstance,
-  publicationContext: emptyPublicationContext,
-};
-
-const emptyPublicationEntityDescription: PublicationEntityDescription = {
-  type: BackendTypeNames.ENTITY_DESCRIPTION,
-  mainTitle: '',
-  abstract: '',
-  description: '',
-  tags: [],
-  npiSubjectHeading: '',
-  date: emptyDate,
-  language: LanguageValues.NONE,
-  contributors: [],
-  isbn: '',
-  numberOfPages: '',
-  series: emptyPublisher,
-  specialization: '',
-  textBook: false,
-  reference: emptyReference,
-};
-
 export type PublicationPreview = Pick<
-  Publication & PublicationEntityDescription,
+  Publication & JournalEntityDescription,
```

Comment:
Denne brukes vel for alle typer, så hvorfor bare Journal her? (samme for `PublishedPublicationPreview`)

Reply:
hvis det bare er reference som er ulik for entityDescription kan vi kanskje flytte resten der til en `BaseEntityDesc` også?

Reply:
Det blir for mye som skal gjøres hvis alt skal fikses i denne PRen. Det er jo forskjellige felt på en publikasjon basert på typer, og da blir naturligvis public publication også forskjellig basert på hvilken type den har. Men jeg kan lage en oppgave på det, sånn at vi ikke glemmer det.

Reply:
https://unit.atlassian.net/browse/NP-1371

Reply:
jaok, så det er foreløpig bare journal som har en "ferdig" public page da?

Reply:
ja 👍